controller/src/main/java/org/jboss/as/controller/parsing/Namespace.java

```diff
@@ -42,53 +42,57 @@
     XML_SCHEMA_INSTANCE("http://www.w3.org/2001/XMLSchema-instance"),
 
     // domain versions, in numerical order
-    DOMAIN_1_0(1, "urn:jboss:domain:1.0"),
+    DOMAIN_1_0(1, 0, "urn:jboss:domain:1.0"),
```

Comment:
Can we please avoid reformatting all of these - it takes a lot of effort porting schema changes between different branches - if we want to fix formatting I think it should be upstream.